stix_shifter_modules/aws_guardduty/configuration/config.json

```diff
@@ -0,0 +1,35 @@
+{
+    "connection": {
+        "type": {
+            "displayName": "AWS GuardDuty",
+            "group": "aws"
+        },
+        "region": {
+            "type": "text",
+            "previous": "connection.options.region"
+        },
+        "help": {
+            "type": "link",
+            "default": "data-sources.html"
+        },
+        "detector_id": {
```

Comment:
why is the `detector_id` user configurable?

Reply:
The code supports two types of input related to detector id.
a. The user can give detector id as input in form of comma separated values.
b. Without any detector id.
The reason for giving detector id as optional configurable, is to support specific detector id search in case if more number of detector ids are available for an account (master account will also list its delegated account's detector ids) in a specific region.